dali/test/python/test_video_pipeline.py

```diff
@@ -90,7 +90,7 @@ def check_videopipeline_supported_type(dtype):
     del pipe
 
 SUPPORTED_TYPES = [types.DALIDataType.FLOAT, types.DALIDataType.UINT8]
-ALL_TYPES = [v for k, v in types.DALIDataType.__dict__.items() if not re.match("__(.*)__", str(k))]
+ALL_TYPES = [v for k, v in types.DALIDataType.__dict__.items() if not (re.match("__(.*)", str(k)) or re.match("name", str(k)))]
```

Comment:
Just use `types.DALIDataType.__members__`
https://pybind11.readthedocs.io/en/stable/classes.html#enumerations-and-internal-types

Reply:
`list(types.DALIDataType.__members__.values())` to be precise

Reply:
Done